### ISSUE TITLE
bioconductor-skeleton computes incorrect checksums for GenomeInfoDbData

### ISSUE THREAD

**Issue by jdblischak** [bioconda/bioconda-utils#236]
While experimenting with the tooling for Bioconductor data packages, I noticed that the checksums for GenomeInfoDbData were incorrect. I re-ran bioconductor-skeleton, but it produced the same incorrect checksums.

```
$ bioconda-utils bioconductor-skeleton . ~/.config/bioconda/conf.yml GenomeInfoDbData
$ grep sha256 bioconductor-genomeinfodbdata/meta.yaml 
  sha256: 39809792f2c88939ed3ea4ecfee1cbdfb1d575194f6a888c2d07e6edf78d3742
$ grep MD5= bioconductor-genomeinfodbdata/post-link.sh 
    MD5="85741ea07f8661079dbd77a23aff3b6b"
$ wget http://bioconductor.org/packages/3.6/data/annotation/src/contrib/GenomeInfoDbData_0.99.1.tar.gz
$ sha256sum GenomeInfoDbData_0.99.1.tar.gz 
80c466e9cdb280dcd0eda4e649f53046a013b204831c03c2684a709b9e5f17f4  GenomeInfoDbData_0.99.1.tar.gz
$ md5sum GenomeInfoDbData_0.99.1.tar.gz 
47c68c7ba79de1c309a5b6009bbc0c71  GenomeInfoDbData_0.99.1.tar.gz
```

However, this is not a consistent problem. For example, `bioconductor-skeleton` produced the correct checksums for SNPlocs.Hsapiens.dbSNP144.GRCh38 in my recent PR https://github.com/bioconda/bioconda-recipes/pull/6959.

```
$ bioconda-utils bioconductor-skeleton . ~/.config/bioconda/conf.yml SNPlocs.Hsapiens.dbSNP144.GRCh38
$ grep sha256 bioconductor-snplocs.hsapiens.dbsnp144.grch38/meta.yaml 
  sha256: b6c6ae8d3a866501146e325d40dc1695b26b522c6f9d063218fc6466bbd5b57b
$ grep MD5= bioconductor-snplocs.hsapiens.dbsnp144.grch38/post-link.sh 
    MD5="128c95e327adf72ae137fb5ae58270fc"
$ wget http://bioconductor.org/packages/3.6/data/annotation/src/contrib/SNPlocs.Hsapiens.dbSNP144.GRCh38_0.99.20.tar.gz
$ sha256sum SNPlocs.Hsapiens.dbSNP144.GRCh38_0.99.20.tar.gz 
b6c6ae8d3a866501146e325d40dc1695b26b522c6f9d063218fc6466bbd5b57b  SNPlocs.Hsapiens.dbSNP144.GRCh38_0.99.20.tar.gz
$ md5sum SNPlocs.Hsapiens.dbSNP144.GRCh38_0.99.20.tar.gz 
128c95e327adf72ae137fb5ae58270fc  SNPlocs.Hsapiens.dbSNP144.GRCh38_0.99.20.tar.gz
```

I've only checked these two Bioconductor data packages, so I don't know if this is an edge case or a widespread problem.

xref: https://github.com/bioconda/bioconda-utils/pull/233 https://github.com/bioconda/bioconda-utils/issues/235

**Comment by mbargull** [comment]
At first I thought this was related to problems regarding the Cargo Port fallback (addressed in #230 and #237), but it looks like the problem is simply that the Bioconductor source for `GenomeInfoDbData` was changed with no version bump 😠 ...

@bgruening: This leads to another problem: The cached fallback tarball from Cargo Port is outdated.
Does Cargo Port have some sort of "revision" mechanism to catch such unfortunate/undesired changes? "Revision" is the sense that it is possible to store not only the oldest (or latest) tarball but allow multiple ones.

**Comment by jdblischak** [comment]
This issue was further investigated in the discussion for PR  #240. @mbargull figured out that the problem was the server cache not being updated https://github.com/bioconda/bioconda-utils/pull/240#issuecomment-351146581 I notified the Bioconductor team in https://github.com/Bioconductor/GenomeInfoDbData/issues/1, and they are working on a new release of the package.

**Comment by jdblischak** [comment]
This has been resolved. The Bioconductor team released 1.0.0 for release 3.6 and this was added to bioconda.

https://www.bioconductor.org/packages/3.6/data/annotation/html/GenomeInfoDbData.html
